docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -21,79 +21,115 @@ To complete this guide, you need:
+----
++
+Some previous releases of GraalVM included the `native-image` tool by default.  This is no longer the case; it must be installed as a second step after GraalVM itself is installed. Note: there is an outstanding issue xref:graal-and-catalina[using GraalVM with macOS Catalina].
+4. (Optional) Set the `JAVA_HOME` environment variable to the GraalVM installation directory.
++
+[source,shell]
+----
+export JAVA_HOME=${GRAALVM_HOME}
+----
+5. (Optional) Add the GraalVM `bin` directory to the path
++
+[source,shell]
+----
+export PATH=${GRAALVM_HOME}/bin:$PATH
+----
 
+[[graal-and-catalina]]
+.Issues using GraalVM with macOS Catalina
 [NOTE]
 ====
-GraalVM binaries are not (yet) notarized for macOS Catalina as reported in this https://github.com/oracle/graal/issues/1724[GraalVM issue].
-To bypass the issue, it is recommended to run the below command instead of disabling macOS Gatekeeper entirely:
+GraalVM binaries are not (yet) notarized for macOS Catalina as reported in this https://github.com/oracle/graal/issues/1724[GraalVM issue]. This means that you may see the following error when using `gu`:
+
+----
+“gu” cannot be opened because the developer cannot be verified
+----
+
+Use the following command to recursively delete the `com.apple.quarantine` extended attribute on the GraalVM install directory as a workaround:
+
 [source,shell]
 -----
-xattr -r -d com.apple.quarantine $HOME/Development/graalvm <1>
+xattr -r -d com.apple.quarantine ${GRAALVM_HOME}
 -----
-
-1. Recursively delete the `com.apple.quarantine` extended attribute on the GraalVM root directory `$HOME/Development/graalvm` we set up above.
-====
-
-[TIP]
-====
-If you cannot install GraalVM, you can use a multi-stage Docker build to run Maven inside a Docker container that embeds GraalVM. There is an explanation of how to do this at the end of this guide.
 ====
 
 == Solution
 
-We recommend that you follow the instructions in the next sections and package the application step by step.
-However, you can go right to the completed example.
+We recommend that you follow the instructions in the next sections and package the application step by step. However, you can go right to the completed example.
 
 Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
 
 The solution is located in the `getting-started` directory.
 
 == Producing a native executable
 
-Let's now produce a native executable for our application.
-It improves the startup time of the application, and produces a minimal disk footprint.
-The executable would have everything to run the application including the "JVM" (shrunk to be just enough to run the application), and the application.
+Let's now produce a native executable for our application. The executable will contain the application, required libraries, the JDK, and an optimized version of a VM. The smaller VM base improves the startup time of the application and produces a minimal disk footprint.
```

Comment:
It won't contain the JDK :).

I would prefer `reduced` or `shrunk` rather than `optimized`. It won't be faster which `optimized` might imply.

Reply:
That is Oracle's language, and I think they mean JDK APIs. Would something like this work:
```
Let's now produce a native executable for our application. The executable will contain
the application, required libraries and Java APIs, and an reduced version of a VM. 
The smaller VM base improves the startup time of the application and produces a 
minimal disk footprint.
```
WDYT ?

Reply:
FYI, a guide should not use the first person ("Let us now"). I prefer "To produce a native executable, ..." or "The next step is to produce a ...."

Reply:
Yeah.. it was there in the original. ;) I can change it, no problem.

Reply:
How about this:
```
The native executable for our application will contain the application code, required libraries, Java APIs, and a reduced version of a VM. The smaller VM base improves the startup time of the application and produces a minimal disk footprint.
```
The steps will just follow. There is no previous step. ;)

Reply:
Yeah, that's OK for me. It's very simplified but it's an introduction guide so let's keep it simple.